samples/synchronousPullWithLeaseManagement.js

```diff
@@ -0,0 +1,126 @@
+// Copyright 2019-2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * This application demonstrates how to perform basic operations on
+ * subscriptions with the Google Cloud Pub/Sub API.
+ *
+ * For more information, see the README.md under /pubsub and the documentation
+ * at https://cloud.google.com/pubsub/docs.
+ */
+
+'use strict';
+
+// sample-metadata:
+//   title: Synchronous Pull With Lease Management
+//   description: Receive messages synchronously,
```

Comment:
The newlines here are interesting.  I wonder if that will have an effect on the generated README?

Reply:
I did some of this in the previous PR actually, and it looked like it was okay. synthtool is pulling these lines out as yaml, and that seems to be pretty standard for multi-line values in yaml. I'll double check it.

Reply:
I re-ran synthtool (I forgot to do that anyway) and it looks good.